crates/symbolicator/src/services/cacher.rs

```diff
@@ -226,7 +227,7 @@ impl<T: CacheItemRequest> Cacher<T> {
     /// # Errors
     ///
     /// If there is an I/O error reading the cache [`CacheItemRequest::Error`] is returned.
-    fn lookup_local_cache(
+    async fn lookup_local_cache(
```

Comment:
I had to look this up, the only really async thing here is an async-aware lock in the shared cache uploader, so that should be fine.